docs/integrations/sources/mongodb-v2.md

```diff
@@ -76,15 +76,21 @@ Make sure that MongoDB is accessible from external servers. Specific commands wi
 
 Your `READ_ONLY_USER` should now be ready for use with Airbyte.
 
+### TLS/SSL on a Connection
+
+It is recommended to use encrypted connection.
+Connection with TLS/SSL security protocol for MongoDb Atlas Cluster and Replica Set instances is enabled by default. Encryption is based on the underlying TLS/SSL support in the JDK.
```

Comment:
suggest removing since it's an implementation detail

```suggestion
Connection with TLS/SSL security protocol for MongoDb Atlas Cluster and Replica Set instances is enabled by default.
```

Reply:
Done